deps/build.jl

```diff
@@ -14,7 +14,18 @@ if is_windows()
     end
 end
 
-libhttp_parser = library_dependency("libhttp_parser", aliases=aliases)
+function validate_httpparser(name,handle)
+    try
+        p = Libdl.dlsym(handle, :http_parser_url_init)
+        return p != C_NULL
+    catch
+        warn("Looks like your binary is old. Please run `rm(\"$(joinpath(dirname(@__FILE__), "usr"))\"; recursive = true)` to delete the old binary and then run `Pkg.build(\"HttpParser\")` again.")
```

Comment:
Instead of quoting it manually and worrying about escaping as @tkelman mentioned, you can just use `show`, e.g. `sprint(show, ....)`.